os/net/mac/csma/csma-output.c

```diff
@@ -85,9 +85,9 @@
 
 /* macMaxFrameRetries: Maximum number of re-transmissions attampts. Range 0--7 */
 #ifdef CSMA_CONF_MAX_FRAME_RETRIES
-#define CSMA_MAX_MAX_FRAME_RETRIES CSMA_CONF_MAX_FRAME_RETRIES
+#define CSMA_MAC_MAX_FRAME_RETRIES CSMA_CONF_MAX_FRAME_RETRIES
 #else
-#define CSMA_MAX_MAX_FRAME_RETRIES 7
+#define CSMA_MAC_MAX_FRAME_RETRIES 7
```

Comment:
maybe `CSMA_MAX_FRAME_RETRIES`?

Reply:
I changed this to the current name to be consistent with TSCH. Tried grepping `MAX_MAC` in CSMA code, found nothing. With your suggest name I would still find nothing...

Reply:
OK I don't really mind about the name but let's keep it consistent with the `_CONF_` version of it

Reply:
Hmm I didn't notice there was already an inconsistency :p would be good to fix anyway, right?